backend/src/main/java/com/google/blackswan/mock/SimpleAlertGenerator.java

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.blackswan.mock;
+
+import com.google.models.*;
+import java.util.*;
+import java.util.Collections;
+
+/** Generate list of alerts by grouping anomalies by their month. */
+public class SimpleAlertGenerator implements AlertGenerator {
+
+  private List<Alert> alerts;
```

Comment:
yep, i fixed it so that function only returns `ImmutableList` and that `Alert`s are part of a `ImmutableList`!

i also changed `List<Anomaly>` in `SimpleAnomalyGenerator` to `ImmutableList<Anomaly>` :)

Reply:
+1 on always thinking about which data structures can be immutable and making it clear in the code if it is an immutable data structure (ie immutable sets, etc) for future PRs :)